packaging/arch/PKGBUILD

```diff
@@ -51,8 +51,10 @@ build() {
 
   # GOFLAGS may be modified by CI tools
   GOFLAGS=""
+  GOFLAGS_SNAP="-tags nomanagers"
   if [[ "$WITH_TEST_KEYS" == 1 ]]; then
       GOFLAGS="$GOFLAGS -tags withtestkeys"
+      GOFLAGS_SNAP="-tags nomanagers,withtestkeys"
```

Comment:
Probably not relevant for arch (because it uses a newer go) - but: go1.9 uses a space separated build tag list. Later go versions use comma separated build tags. The docs on this are hard to find, "go help build" gives the details. Mostly for reference.